p2p/discover/v5_udp.go

```diff
@@ -164,7 +164,7 @@ func newUDPv5(conn UDPConn, ln *enode.LocalNode, cfg Config) (*UDPv5, error) {
 		closeCtx:       closeCtx,
 		cancelCloseCtx: cancelCloseCtx,
 	}
-	tab, err := newTable(t, t.db, cfg.Bootnodes, cfg.Log)
+	tab, err := newTable(t, t.db, cfg.Bootnodes, cfg.Log, nil)
```

Comment:
Why  in `v4_udp.go` `cfg.FilterFunction` is passed, whereas here `nil` is passed?

Reply:
It makes sense to add it I think. By default it should be `nil` anyway.